__tests__/app.test.js

```diff
@@ -49,4 +49,33 @@ describe("NC News API testing", () => {
       });
     });
   });
+  describe("/api/articles/:article_id", () => {
+    test("return correct article when passed an article id ", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+          expect(Object.keys(body.articles)).toHaveLength(8);
+        });
```

Comment:
tests should include following - 
        to allow for later extension of the endpoint, the test should use a flexible matcher eg expect.objectContaining or .toMatchObject.
    Status 400, invalid ID, e.g. string of "not-an-id"